richardwellerson/style.css

```
@@ -0,0 +1,211 @@
+* {
+    cursor: url("images/cursor.png"), auto;
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0 auto;
+    background: #dddddd;
+    word-break: normal;
+    text-align: justify;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    background-color: black;
+    color: #dddddd;
+    padding: 1%;
+    border: 2px solid whitesmoke;
+}
+
+a {
+    max-width: 100%;
+}
+
+header {
+    padding: 5%;
+    width: 100%;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    word-break: normal;
+}
+
+.star {
+    width: 5%;
+    animation: 0.5s infinite;
+    animation-name: star-rotate;
```

Aqui dá pra simplificar assim:

```suggestion
animation: star-rotate 0.5s infinite;
```

---

resolvido, Alberto!